### PR TITLE
Notes grid clickable cards & QoL

### PR DESCRIPTION
**Pull request by lusicle**
# Description

- [x] Notes grid now has clickable cards
- [x] Notes cards open modal views for plain/secret/wifi notes
- [x] Notes modal view now uses the note title as the heading for the view
- [x] Flatshare logo now returns users to the dashboard

Part of #85 & General QoL

## Type of change

- [x] **New feature** (non-breaking change which adds functionality)
- [x] **Improvement** (non-breaking change which improves existing functionality)

# Checklist:

I have completed these steps when making this pull request:

- [x] I have assigned my name to the issue
- [x] I have moved the issue to the **In Progress** column
- [x] I have labelled the PR appropriately
- [x] I have assigned myself to the PR

Before opening the PR for review:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] I have added attributions to new dependencies and resources

### CODE REVIEW

File: packages/frontend/src/components/notes/note/NotesGrid.tsx [hunk 1]
```diff
@@ -13,12 +16,76 @@ interface NotesGridProps {
...
+  const [wifivisible, setWifiVisible] = useState(false);
+  const [wifipass, setWifiPass] = useState('');
+  const [wifiuser, setWifiUser] = useState('');
+  const [activeName, setActiveName] = useState('');
+  const [activeValue, setActiveValue] = useState('');
+  const [qrCodeText, setQrCodeText] = useState('');
+  const [qrvisible, setQRVisible] = useState(false);
+  const [loading] = useState(false);
+  const onClickCard = (item: any) => {
+    // NEED TO CHANGE SO TYPES WORK PROPERLY
```
Comment (dkap937-209) [inline packages/frontend/src/components/notes/note/NotesGrid.tsx:30]: What changes need to be made? What is it currently doing?

**Review by dkap937-209 — APPROVED** [review]
LGTM

File: packages/frontend/src/components/notes/note/NotesGrid.tsx (hunk 1 above)
Comment (lusicle) [reply packages/frontend/src/components/notes/note/NotesGrid.tsx:30]: Ah I need to remove that comment, good spot.

**Review by rushaan01 — APPROVED** [review]
LGTM

**Pull request merged**